### PR TITLE
TIA rebase changes

### PR DESCRIPTION
**Pull request by kaadhina**

### CODE REVIEW

**Comment by acesiddhu** [comment]
>             "name": "testFiltercriteria",

can u one more change here? can u add a checkbox with TIA on/off and a textbox that take rebase limit? lets read the values from that instead starting 106

---
Refers to: Tasks/VsTest/task.json:50 in 45e7456. [](commit_id = 45e7456e30fc28bf5fcc0c9502159db2d78426d5, deletion_comment = False)

File: Tasks/VsTest/vstest.ts [hunk 1]
```diff
@@ -36,6 +36,7 @@ try {
     var runInParallel: boolean = tl.getBoolInput('runInParallel');
     var tiaEnabled = tl.getVariable('tia.enabled');
     var fileLevel = tl.getVariable('tia.filelevel');
+    var tiaRebaseLimit = tl.getVariable('tia.rebaselimit');
```
Comment (acesiddhu) [inline Tasks/VsTest/vstest.ts]: lets start adding options in the task ui itslef. check the story board in PPT

**Comment by acesiddhu** [comment]
:shipit:

File: Tasks/VsTest/vstest.ts (hunk 1 above)
Comment (kaadhina) [inline Tasks/VsTest/vstest.ts]: Will take this in another PR, since there are some open questions for UI. Pushing in the remaining for now.

**Pull request merged**